Chinese/CrazyStone/CrazyStone.md

```diff
@@ -2,4 +2,4 @@
 他打算挖地洞进入藏宝地点，说起挖地洞，现在问题来了。
 他没学过挖洞的技能，况且手头也没有工具，独自一人徒手挖洞显然无法想象。正在他抓耳挠腮绞尽脑汁的紧要关头，上天灵光突然照亮了他的脑袋---要挖洞，找蓝翔！！！
 麦克二话不说，径直奔向重庆火车站。此时正值春运，麦克排了三天三夜，终于轮到自己却被告知开往山东的火车票都已售罄……麦克心灰意冷地走出火车站，却意外撞见了小贼中的一名，原来他也没买到去蓝翔的火车票。两人一阵唏嘘，
-相见恨晚，大排档喝到半夜。第二天早上6点麦克被广场舞震醒，发现自己一个人躺在床上。
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.